common/attest_plugin.c

```diff
@@ -417,6 +417,60 @@ oe_result_t oe_verify_attestation_certificate_with_evidence(
     size_t cert_in_der_len,
     oe_verify_claims_callback_t claim_verify_callback,
     void* arg)
+{
+    oe_result_t result = OE_FAILURE;
+    oe_claim_t* claims = NULL;
+    size_t claims_length = 0;
+
+    result = oe_verify_attestation_certificate_with_evidence_v2(
+        cert_in_der,
+        cert_in_der_len,
+        NULL,
+        0,
+        NULL,
+        0,
+        &claims,
+        &claims_length);
+
+    if (result != OE_OK)
+        OE_RAISE_MSG(
+            result,
+            "oe_verify_attestation_certificate_with_evidence() failed with "
+            "error = %s\n",
+            oe_result_str(result));
+
+    //---------------------------------------
+    // call client to further check claims
+    // --------------------------------------
+    if (claim_verify_callback)
+    {
+        result = claim_verify_callback(claims, claims_length, arg);
+        OE_CHECK(result);
+        OE_TRACE_VERBOSE("claim_verify_callback() succeeded");
+    }
+    else
+    {
+        OE_TRACE_WARNING(
+            "No claim_verify_callback provided in "
+            "oe_verify_attestation_certificate_with_evidence call",
+            NULL);
+    }
+
+done:
+    if (claims)
+        oe_free_claims(claims, claims_length);
+    return result;
+}
+
+oe_result_t oe_verify_attestation_certificate_with_evidence_v2(
+    uint8_t* cert_in_der,
+    size_t cert_in_der_len,
```

Comment:
```suggestion
    size_t cert_in_der_size,
```

avoid abbreviations per OE style guidelines